src/urh/dev/native/lib/bladerf.pyx

```diff
@@ -129,8 +129,23 @@ cpdef int set_center_freq(bladerf_frequency frequency):
     return bladerf_set_frequency(_c_device, get_current_bladerf_channel(), frequency)
 
 cpdef bladerf_frequency get_center_freq():
-    cdef bladerf_frequency result = 0
-    err = bladerf_get_frequency(_c_device, get_current_bladerf_channel(), &result)
+  cdef bladerf_frequency result = 0
+  err = bladerf_get_frequency(_c_device, get_current_bladerf_channel(), &result)
+
+  if err != 0:
+      return 0
+
+  return result
+
+cpdef int set_bias_tee(on_or_off):
+  IF BLADERF_API_VERSION >= 2:
+    cdef bool bias_tee = 1 if on_or_off else 0
+    return bladerf_set_bias_tee(_c_device, get_current_bladerf_channel(), bias_tee)
+
+cpdef int get_bias_tee():
+  IF BLADERF_API_VERSION >= 2:
```

Comment:
Can we add a return value for the ELSE case? I am not sure if Cython likes empty method definitions

Reply:
Added.